src/components/notifications/notifications.util.tsx

```diff
@@ -358,3 +353,8 @@ export const renderErrorToast = () =>
         variant: ToastVariantType.error,
         description: 'Some required fields are missing or Invalid',
     })
+
+export const getAwsRegionListParsed = (awsRegionList) =>
```

Comment:
This should be removed

Reply:
We are using this util in other component

Reply:
Moved this util to the component.